storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -1672,7 +1703,7 @@ private int fragmentedCpu() {
         long beforeSchedule = System.currentTimeMillis();
         scheduler.schedule(topologies, cluster);
         long scheduleTimeElapsedMs = System.currentTimeMillis() - beforeSchedule;
-        LOG.info("Scheduling took {} ms for {} topologies", scheduleTimeElapsedMs, topologies.getTopologies().size());
+        LOG.debug("Scheduling took {} ms for {} topologies", scheduleTimeElapsedMs, topologies.getTopologies().size());
```

Comment:
It doesn't look like too verbose even if it is info. Do you have specific reason to change?

Reply:
yes, it is too verbose, if we want a monitor of scheduling, we should make a metric for it but not through logging

Reply:
Makes sense.